float64 percent_camera
float64[] coords_camera
string image_camera
string image_radar
string direction
